client/src/api/index.ts

```diff
@@ -183,7 +183,7 @@ export const getChatRooms = async (userDocumentId: string) => {
 export const getFollowingsList = async (userDocumentId: string) => {
   try {
```

Comment:
fetch 부분에서 error가 발생하는 경우가 있어서 try catch문으로 감싸셨군요!

아마 400번대, 500번대 에러 그리고 type 에러를 잡기 위해서 다음과 같이 구현하셨을것같아요.

https://developer.mozilla.org/en-US/docs/Web/API/fetch#examples

> Fetch promises only reject with a TypeError when a network error occurs. Since 4xx and 5xx responses aren't network errors, there's nothing to catch. You'll need to throw an error yourself to use Promise#catch.

400, 500번대 응답의 경우는 네트워크 에러로 취급하지 않아 catch문에서 잡히지 않을것 같은데요, 한번 확인해보시면 좋을것같습니다.